src/app/comic/comic-list-container/comic-list-container.component.ts

```diff
@@ -33,4 +42,31 @@ export class ComicListContainerComponent implements OnInit {
     )
   }
 
+  onComicModalClick() {
+    this.showComicModal = true;
+  }
+
+  hideModal() {
+    this.showComicModal = false;
+  }
+
+  @HostListener('document:click', ['$event'])
+  clickOnWindow(event: any) {
+    if (event.target.className == "card-container" || event.target.className === "card") {
```

Comment:
not needed to hide the modal on card click..

Reply:
Okay